src/history/hs.c

```diff
@@ -1808,32 +1808,38 @@ __verify_history_store_id(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt, uint32
 int
 __wt_history_store_verify_one(WT_SESSION_IMPL *session)
 {
-    WT_CURSOR *cursor;
-    WT_CURSOR_BTREE cbt;
+    WT_CURSOR *hs_cursor;
+    WT_CURSOR_BTREE ds_cbt;
     WT_DECL_RET;
     WT_ITEM hs_key;
     uint32_t btree_id;
     int exact;
 
-    cursor = session->hs_cursor;
+    hs_cursor = session->hs_cursor;
     btree_id = S2BT(session)->id;
 
     /*
      * We are required to position the history store cursor. Set it to the first record of our btree
      * in the history store.
      */
     memset(&hs_key, 0, sizeof(hs_key));
-    cursor->set_key(cursor, btree_id, &hs_key, 0, 0);
-    ret = __wt_hs_cursor_search_near(session, cursor, &exact);
+    hs_cursor->set_key(hs_cursor, btree_id, &hs_key, 0, 0);
+    ret = __wt_hs_cursor_search_near(session, hs_cursor, &exact);
     if (ret == 0 && exact < 0)
-        ret = __wt_hs_cursor_next(session, cursor);
+        ret = __wt_hs_cursor_next(session, hs_cursor);
 
-    /* If we positioned the cursor there is something to verify. */
+    /*
```

Comment:
Added comment to explain why we use `__wt_btcur_open ` instead of opening a standard cursor.